@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/core/fn_coreScheduler.sqf

```diff
@@ -37,6 +37,7 @@ if ((diag_ticktime - fza_ah64_overalltickerSlow) > _tickerSlow && alive _heli &&
         [_heli] call _x;
     }
     foreach fza_ah64_slowschedarray;
+    [_heli] spawn fza_fnc_fcrLongbow;
```

Comment:
Why not just put it in the slow scheduler?  If you remove the sleep then it can become unscheduled

Reply:
removing the sleep kills the fps to 10ish